test/SlowTests/Server/Documents/ETL/ConnectionStringTests.cs

```diff
@@ -39,6 +40,15 @@ public void CanAddAndRemoveConnectionStrings()
 
                 var result1 = store.Maintenance.Send(new PutConnectionStringOperation<SqlConnectionString>(sqlConnectionString));
                 Assert.NotNull(result1.RaftCommandIndex);
+                
+                var elasticsearchConnectionString = new ElasticsearchConnectionString
+                {
+                    Name = "ElasticsearchConnectionString",
+                    Nodes = new[]{"http://127.0.0.1:8080" },
```

Comment:
We usually use `8080` for RavenDB, which can be confusing.

Reply:
I will change that. The default for Elasticsearch is 9200